service/src/main/java/cz/muni/fi/pa165/facade/ManufacturerFacadeImpl.java

```diff
@@ -0,0 +1,70 @@
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.dto.BottleDTO;
+import cz.muni.fi.pa165.dto.ManufacturerDTO;
+import cz.muni.fi.pa165.entity.Manufacturer;
+import cz.muni.fi.pa165.service.BeanMappingService;
+import cz.muni.fi.pa165.service.ManufacturerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.inject.Inject;
+import java.util.Collection;
+import java.util.Date;
+
+/**
+ * @author Martin Sumera
+ */
+@Service
+@Transactional
+public class ManufacturerFacadeImpl implements ManufacturerFacade {
+
+    @Inject
+    private ManufacturerService manufacturerService;
+
+    @Inject
+    private BeanMappingService beanMappingService;
+
+    @Override
+    public void createManufacturer(ManufacturerDTO manufacturerDTO) {
+        Manufacturer manufacturer = beanMappingService.mapTo(manufacturerDTO, Manufacturer.class);
+        manufacturerService.createManufacturer(manufacturer);
+    }
+
+    @Override
+    public Collection<ManufacturerDTO> findAll() {
+        Collection<Manufacturer> manufacturers = manufacturerService.findAll();
```

Comment:
Pouzil bych List misto Collection. Az to budeme zobrazovat na webu, tak bychom urcite chteli, aby se nam ty polozky zobrazovaly v nejakem poradi a ne pri kazdem refreshi jinak.